src/core/externalstorage/qgsexternalstorage.h

```diff
@@ -157,8 +157,17 @@ class CORE_EXPORT QgsExternalStorageContent : public QObject
      */
     void reportError( const QString &errorMsg );
 
-    Qgis::ContentStatus mStatus = Qgis::ContentStatus::NotStarted;
+    /**
+     * Sets the external storage status
```

Comment:
```suggestion
     * Sets the external storage \a status
```
We need to document all parameter. Strange that CI doesn't warn about this